coordinator/src/services/translation/index.ts

```diff
@@ -104,9 +103,10 @@ function createParametersDisplay(fragmentsToDisplay: Display): string {
   return Buffer.from(Mustache.render(displayTemplate, fragmentsToDisplay)).toString("base64")
 }
 
-function createParameters(base64Payload: string, base64Display: string): fhir.Parameters {
+function createParameters(base64Fragments: string, base64Digest: string, base64Display: string): fhir.Parameters {
   const parameters: Array<fhir.Parameter> = []
-  parameters.push({name: "payload", valueString: base64Payload})
+  parameters.push({name: "fragments", valueString: base64Fragments})
```

Comment:
The clearest name for this parameter is `fragments`, but calling it `payload` would mean the Signing Service would accept it unchanged